calculate.py

```diff
@@ -30,12 +31,15 @@ def evaluate(expression: str, is_rad: bool) -> str:
         # Convert numbers directly to rad
         if exp_is_only_num(expression):
             if is_rad == "true":
-                expression = str(display.rad((float)(expression)))
+                expression = str(display.rad(float(expression)))
             elif is_rad == "false":
-                expression = str(display.deg((float)(expression)))
+                expression = str(display.deg(float(expression)))
 
         # Evaluate expression
-        return parser.evaluate(expression)
+        evaluation = parser.evaluate(expression)
+        if is_binary == "true":

```

Comment:
Why are we comparing the value of is_binary to a string? Why not just is_binary == true?

Reply:
Having read Swati's PR and seen the code below, I now see that this variable is in fact a string. That being said, the method signature should be modified to reflect that.

Reply:
The method signature of which function? Currently in evaluate() it's: 
def evaluate(self: object, expression: str) -> float:

Reply:
I meant the type hinting for is_binary. The type is listed as boolean but it's actually a string.

Reply:
Thanks I updated this